mods/ra/weapons/ballistics.yaml

```diff
@@ -253,28 +159,20 @@ Grenade:
 			Light: 25
 			Heavy: 25
 			Concrete: 100
-		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Tank, Structure, Wall, Husk, Trees
 	Warhead@3Eff: CreateEffect
 		Explosions: med_explosion
 		ImpactSounds: kaboom25.aud
-		InvalidImpactTypes: Water
 	Warhead@4EffWater: CreateEffect
 		Explosions: small_splash
-		ImpactSounds: splash9.aud
-		ValidImpactTypes: Water
 
 DepthCharge:
+	Inherits: ^Artillery
```

Comment:
This introduces a new (artillery) firing sound when destroyers fire on subs.  It feels a little out of place considering the very slow flight rate of the charge.